bucket/devspace.json

```diff
@@ -0,0 +1,46 @@
+{
+    "version": "6.3.2",
+    "description": "DevSpace is an open-source developer tool for Kubernetes that lets you develop and deploy cloud-native software faster",
+    "homepage": "https://devspace.sh",
+    "license": "Apache-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/devspace-sh/devspace/releases/download/v6.3.2/devspace-windows-amd64.exe",
+            "hash": "72d89cd7f2c4b59a4ae282293423318eb5cac68a1bb48dcc50ff200f3ce826a6",
+            "bin": [
```

Comment:
```suggestion
```

...so you can use single `binary` entry below instead.

Reply:
done in https://github.com/ScoopInstaller/Main/pull/4737/commits/50e1e6f52444ec3a5892c7c3467bb3d5d0933724